src/main/resources/db/migration/V4__Create_Cart_and_Customer_Tables.sql

```diff
@@ -1,21 +1,21 @@
--- إنشاء جدول الزبائن (customers) إذا لم يكن موجودًا
+-- V4__Create_Cart_and_Customer_Tables.sql
+-- Create the customers table if it does not exist
```

Comment:
Migration files should not be updated, create a new migration file to alter table structure instead